GitHubVS.sln

```diff
@@ -1,11 +1,14 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27703.2035
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28516.95
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GitHub.VisualStudio", "src\GitHub.VisualStudio\GitHub.VisualStudio.csproj", "{11569514-5AE5-4B5B-92A2-F10B0967DE5F}"
 	ProjectSection(ProjectDependencies) = postProject
+		{161DBF01-1DBF-4B00-8551-C5C00F26720D} = {161DBF01-1DBF-4B00-8551-C5C00F26720D}
+		{161DBF01-1DBF-4B00-8551-C5C00F26720E} = {161DBF01-1DBF-4B00-8551-C5C00F26720E}
```

Comment:
You can remove these solution dependencies if the `ReferenceOutputAssembly=false` metadata trick works.